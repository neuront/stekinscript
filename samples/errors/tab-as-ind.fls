if 0
	write(0)
else
  	write(1)

ifnot (0 = 1)
	 write(0)
