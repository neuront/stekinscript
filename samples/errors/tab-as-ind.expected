Line: 2
    use tab as indent is forbidden.
Line: 4
    use tab as indent is forbidden.
Line: 3
    `else' does not match an `if'.
Line: 7
    use tab as indent is forbidden.
